client.go

```diff
@@ -294,6 +295,27 @@ func (c *Client) SetAuthToken(token string) *Client {
 	return c
 }
 
+// SetAuthScheme method sets the auth token scheme type in the HTTP request. For Example:
```

Comment:
s/SetAuthScheme/The SetAuthScheme/
s/token//

I'd also reword this to be like my wordsmithing suggestion on SetAuthToken.

Reply:
Method name has to start first and the `token` part taken care in my branch will push it to rigrassm:master